atqa-framework/utils/fileUpload.ts

```diff
@@ -0,0 +1,11 @@
+import { File } from '../interfaces/File.interface';
+import { Locator } from '../interfaces/Locator.interface';
+
+const { I } = inject();
+
+export const uploadFile = async (
```

Comment:
Doesn't seem to be used

Reply:
same as above

Reply:
You can keep it in a separate branch then